packages/contracts-bedrock/deploy-config/goerli.json

```diff
@@ -1,36 +1,45 @@
 {
-  "finalSystemOwner": "DUMMY",
-  "controller": "DUMMY",
+  "numDeployConfirmations": 1,
+
+  "finalSystemOwner": "0xBc1233d0C3e6B5d53Ab455cF65A6623F6dCd7e4f",
+  "controller": "0xBc1233d0C3e6B5d53Ab455cF65A6623F6dCd7e4f",
```

Comment:
These don't match the addresses in the notion table, did we change keys?

Reply:
Yes - upon further discussion we set this to a multisig to match mainnet.

Reply:
noting that having a multisig as the controller will make the upgrade slower than our rehearsals, but I like that it's closer to 1:1 with the process we will have on mainnet